modules/backing-up-red-hat-quay-operator.adoc

```diff
@@ -76,6 +76,13 @@ All information under the `data` property should remain the same.
 $ oc get secret -n <quay-namespace>  $(oc get quayregistry <quay-registry-name> -n <quay-namespace>  -o jsonpath='{.spec.configBundleSecret}') -o yaml > config-bundle.yaml
 ----
 
+. Backup the `/conf/stack/config.yaml` file mounted inside of the Quay pods:
++
+[source,terminal]
+----
+$ oc exec -it quay-pod-name -- cat /conf/stack/config.yaml > quay-config.yaml
+----
+
```

Comment:
In fact, we don't need to run these new lines.  

The `config.yaml` has been included in the secret backed up in original line 76.  

When we restore this secret,  we have restore the `config.yaml` file.

Reply:
@huweihua-redhat that's not correct. Line 76 does not back up the rendered config.yaml file. It actually backs up the custom config bundle that is being fed to the operator. The rendered config.yaml file contains much more information than the custom config bundle does. Custom config bundle does not contain `SECRET_KEY` and `DATABASE_SECRET_KEY` which cannot be random once the registry is installed, for example. And those two values are needed during restoring, otherwise we cannot decrypt various tokens stored in the database. The custom config bundle is useful only when initially installing the cluster. Custom config bundle is not useful that much when restoring an already installed cluster.

Reply:
We specifically pened this doc bug because we've had a client try the original approach and it partially broke their installation.

Reply:
`managed-secret-keys` exist only on Quay 3.7. The backup and restore procedures IMHO should be agnostic to the version of Quay.

Reply:
@ibazulic thanks for your information. 

For quay 3.7 and eariler version, I need time to double check these steps in a real environment before I can give you the final confirmation. But Quay QE team are busy working on the other high priority items recently.   I have to start this double-check job one week later. Is that ok for you?

Another thing is that even we decide to add these steps finally, in my opinion, we need to update the restore steps accordingly. Otherwise, customer won't know how to import this `config.yaml`.  So the changes in this PR are not complete. I still need time to find out the easy and suitable restore steps for this backup change.

So,  is that ok for you to wait for a few days?   thanks a lot.

Reply:
@huweihua-redhat this has been escalated by Claire Davey. Any update? Should I merge?

Reply:
I don't know what is there to check. The notes are mostly based on my own backup/restore procedures anyway. We need the procedure to be as agnostic as possible. Not everyone is runing a 3.7 installation. I also don't understand what needs to be tested here. You literary grab *one more* file, that's it. We can add the disclaimer that this step is necessary if a client is ruuning Quay version 3.6 or below, but it needs to be added. It can be removed once we deprecate versions 3.5 and 3.6.

Reply:
Ok, we can merge it first.  The changes is errorless and harmless even it is not 100% perfect. We can refine the rest  later.  thanks @stevsmit @ibazulic 
LGTM